install.sh

```
@@ -2868,6 +3887,39 @@ start_birdnet_go
+# Log final system state  
+log_system_resources "post-install"
+log_docker_state "post-install"  
+log_service_state "post-install"
+
+# Log final config file hash
+if [ -f "$CONFIG_FILE" ]; then
+    log_config_hash "final"
+fi
+
+# Verify service is responding
+local final_service_responsive="false" 
```

Shellcheck (0.10.0)

[error] 'local' is only valid in functions.

(SC2168)